app/src/main/java/org/mozilla/fenix/components/BackgroundServices.kt

```diff
@@ -142,7 +125,12 @@ class BackgroundServices(
             context = context,
             service = pushService,
             config = pushConfig
-        )
+        ).also {
```

Comment:
Wondering if I should remove this right now, and land it in a follow-up PR when we remove the same code in the App class.

Reply:
We removed it. Will land when we do the last bit for perf.